iina.xcodeproj/project.pbxproj

```diff
@@ -2037,7 +2037,7 @@
 				sk,
 				sv,
 				ro,
-        hi,
+        		hi,
```

Comment:
What did you do in this commit? It seems not right viewing on GitHub.

Reply:
When I pressed resolve conflict issues on Github Web, It doesn't indents properly according to Xcode indentation. So I had to take pull on my local machine & add proper indentation in "project.pbxproj" file using Visual Studio Code editor.

Reply:
But still, it looks not correct now. Can you try to fix that again?

Reply:
So on github Web it shows different indentation, should I just revert my indentation changes ?

Reply:
Attaching a screenshot :) 
On github Desktop & Xcode / VS Code looks fine. 
<img width="1187" alt="screenshot 2019-01-31 at 20 45 07" src="https://user-images.githubusercontent.com/8469205/52063743-30c5bc80-2599-11e9-86cc-6ab20a51d385.png">

Reply:
Your previous version used all spaces; current version used mixed spaces and tabs. Only tabs is used in other lines.

Reply:
Please suggest me what should I use to check whether the indentation is correct. 

Now using VS Code for deleting whitespace / Tabs & then pressing enter.

<img width="1067" alt="screenshot 2019-01-31 at 21 05 37" src="https://user-images.githubusercontent.com/8469205/52065189-00cbe880-259c-11e9-9029-e4111dc509fa.png">